Topics/Tech_Stacks/Django_Rest.md

```diff
@@ -109,7 +132,7 @@ urlpatterns = [
 ]
 ```
 
-All in all, we have now built a simple API with the ability to list, create, update and delete todo list items. One can test the API by starting the Django development server using ``` python3 manage.py runserver ``` and visiting http://localhost:8000/todos/ in a web browser.
+All in all, we have now built a simple API with the ability to list, create, update and delete todo list items. One can test the API by starting the Django development server using ``` python3 manage.py runserver ``` and visiting http://localhost:8000/todos/ in a web browser. Running this command will also show any errors that need to be fixed in the shell, and this continuously updates whenever the code is changed.
```

Comment:
Can you change the last line to indicate what continuously updates? I'm not sure what "this" is.

Reply:
[ea08aee](https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/310/commits/ea08aee1dd22045fff731064ae7c678a898aa516)